PublicMapping/districtbuilder
# Make paintbrush selection tool configurable by slider

## Overview

From #825 :

- There is a slider that's styled
- The slider will be toggled appropriately based on selection tool
- The slider remembers the size you picked last
- A slider circle is shown on the page that corresponds to the size you picked (NOTE: This is not specified as required in the issue, but I think we need to somehow visually indicate the paintbrush area as you're using it)

Newly added:

- Paintbrush selection tool selects all elements within brush area for all radii

### Checklist




### Notes

There are some scenarios where certain geounits are not selected at certain zoom levels, likely due to tile layer issues. Thus far, I have only observed this happening with Dauphin County, PA at high zoom levels. It feels a bit wonky, but since you can select the areas fine by zooming in a bit it doesn't seem to be a _huge_ problem - but potentially it is something worth looking into a little further

![image](https://user-images.githubusercontent.com/66973361/123171684-20545c00-d44a-11eb-8d1c-4de80d814627.png)


## Testing Instructions

- Select the paintbrush tool and the slider should show
- Default should be 1
- Select a different size
- Choose a different selection tool and the slider should be hidden
- Choose the paintbrush tool again and it should have remembered your size from before
- Click and hold and drag to select features on the map and you should see a partially opaque circle which corresponds to the size you choose
- All features under the paintbrush tool are selected as you drag, not just what's directly intersecting with your cursor

Closes #742
